processing/topics/sidepath_estimation/sidepath_estimation.sql

```diff
@@ -0,0 +1,26 @@
+\set QUIET on
+\set ON_ERROR_STOP on
+
+-- set parameter defaults
+\set buffer_size 22.0
+\set buffer_distance 100.0
+\set outfile /data/hashes/is_sidepath.idlines.txt
+   
+-- reset sequence for checkpoint numbering
+\o /dev/null
+SELECT setval('checkpoint_nr_sequence', 1);
+
+-- set output to the result file
+\o :outfile
+
+\echo `date` 'Start generating is_sidepath ids'
+
+-- set appropiate formatting 
+\pset format unaligned
+\pset tuples_only on
+
+-- run the query
+SELECT * FROM sidepath_idlist_yes(:buffer_distance, :buffer_size);
```

Comment:
Using `sidepath_idlist_no` produces significantly smaller output (6MiB vs 30MiB) but is also significantly slower (15min vs 30min) with the current implementation